src/test/anovos/data_analyzer/test_association_evaluator.py

```diff
@@ -355,196 +357,65 @@ def test_variable_clustering(spark_session: SparkSession):
-        == 28
-    )
-    assert (
-        test_df.where(F.col("ifa") == "4062a").toPandas().to_dict("list")["sex"][0]
-        == "Male"
-    )
-    assert (
-        test_df.where(F.col("ifa") == "4062a")
-        .toPandas()
-        .to_dict("list")["education"][0]
-        == "11th"
-    )
 
-    result_df3 = correlation_matrix(spark_session, test_df, drop_cols=["ifa"])
+    num_cols, cat_cols, other_cols = attributeType_segregation(test_df)
+
+    result_df3 = correlation_matrix(
+        spark_session, test_df, list_of_cols=num_cols, drop_cols=[]
+    )
 
-    assert result_df3.count() == 15
+    assert result_df3.count() == 7
     assert (
         result_df3.where(F.col("attribute") == "age")
         .toPandas()
         .to_dict("list")["age"][0]
         == 1.0
     )
-    assert (
-        result_df3.where(F.col("attribute") == "age")
-        .toPandas()
-        .to_dict("list")["native-country"][0]
-        <= 0.2
-    )
     assert (
         result_df3.where(F.col("attribute") == "age")
         .toPandas()
         .to_dict("list")["capital-gain"][0]
-        <= 0.25
+        <= 0.1
     )
     assert (
         result_df3.where(F.col("attribute") == "age")
         .toPandas()
         .to_dict("list")["capital-loss"][0]
-        <= 0.25
-    )
-    assert (
-        result_df3.where(F.col("attribute") == "age")
-        .toPandas()
-        .to_dict("list")["education"][0]
-        <= 0.4
+        <= 0.1
     )
     assert (
         result_df3.where(F.col("attribute") == "age")
         .toPandas()
         .to_dict("list")["education-num"][0]
-        <= 0.4
-    )
-    assert (
-        result_df3.where(F.col("attribute") == "age")
-        .toPandas()
-        .to_dict("list")["fnlwgt"][0]
         <= 0.1
     )
     assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
+        result_df3.where(F.col("attribute") == "education-num")
         .toPandas()
         .to_dict("list")["capital-gain"][0]
-        == 1.0
-    )
-    assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
-        .toPandas()
-        .to_dict("list")["native-country"][0]
-        <= 0.1
-    )
-    assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
-        .toPandas()
-        .to_dict("list")["age"][0]
         <= 0.2
     )
     assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
+        result_df3.where(F.col("attribute") == "education-num")
         .toPandas()
         .to_dict("list")["capital-loss"][0]
         <= 0.1
     )
     assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
+        result_df3.where(F.col("attribute") == "fnlwgt")
         .toPandas()
-        .to_dict("list")["education"][0]
-        <= 0.3
+        .to_dict("list")["logfnl"][0]
+        <= 0.95
     )
     assert (
-        result_df3.where(F.col("attribute") == "capital-gain")
+        result_df3.where(F.col("attribute") == "hours-per-week")
```

Comment:
why did you change the column names here?

Reply:
these test cases were not added by me, may be @ChandraShekhar2 ?